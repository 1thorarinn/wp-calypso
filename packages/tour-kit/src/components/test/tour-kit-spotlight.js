/**
 * @jest-environment jsdom
 */
import { shallow } from 'enzyme';
import TourKitOverlay from '../tour-kit-overlay';
import TourKitSpotlight from '../tour-kit-spotlight';

describe( 'TourKitSpotlight', () => {
	test( 'should render', () => {
		const wrapper = shallow( <TourKitSpotlight /> );

		expect( wrapper ).toMatchSnapshot();
	} );

	test( 'should render Overlay visible when no referenceElement', () => {
		const wrapper = shallow( <TourKitSpotlight /> );
		const overlay = wrapper.find( TourKitOverlay );
		expect( overlay.prop( 'visible' ) ).toBe( true );
	} );
} );
