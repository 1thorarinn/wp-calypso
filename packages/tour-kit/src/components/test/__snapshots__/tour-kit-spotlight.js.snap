// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TourKitSpotlight should render 1`] = `
<Fragment>
  <TourKitOverlay
    visible={true}
  />
  <div
    className="tour-kit-spotlight"
  />
</Fragment>
`;
